server/odc-service/src/main/java/com/oceanbase/odc/service/session/OBExecutionListener.java

```diff
@@ -31,16 +31,21 @@
 import com.oceanbase.odc.core.session.ConnectionSessionUtil;
 import com.oceanbase.odc.core.sql.execute.model.JdbcGeneralResult;
 import com.oceanbase.odc.core.sql.execute.model.SqlTuple;
-import com.oceanbase.odc.core.sql.parser.AbstractSyntaxTree;
 import com.oceanbase.odc.core.sql.util.OBUtils;
 import com.oceanbase.odc.service.queryprofile.OBQueryProfileManager;
 import com.oceanbase.odc.service.session.model.AsyncExecuteContext;
+import com.oceanbase.tools.dbbrowser.parser.ParserUtil;
+import com.oceanbase.tools.dbbrowser.parser.constant.GeneralSqlType;
 import com.oceanbase.tools.dbbrowser.parser.constant.SqlType;
+import com.oceanbase.tools.dbbrowser.parser.result.BasicResult;
+
+import lombok.extern.slf4j.Slf4j;
 
 /**
  * @author: liuyizhuo.lyz
  * @date: 2024/4/23
  */
+@Slf4j
```

Comment:
we may change another name to tell the coder that this listener is only used for query profile~

Reply:
renamed to `OBQueryProfileExecutionListener`